cl/lib/search_utils.py

```diff
@@ -989,16 +989,21 @@ def get_related_clusters_with_cache(cluster, request):
         for item in sub_opinion_queries:
             sub_opinion_query |= item
 
+        # Set MoreLikeThis parameters
+        # (see https://lucene.apache.org/solr/guide/6_6/other-parsers.html#OtherParsers-MoreLikeThisQueryParser)
+        mlt_params = dict(
```

Comment:
Why use dict here? Generally it's better just to use `{}`.

Reply:
I was just too lazy to write the `"` ;-)